packages/components/src/FormatFile/FormatFile.test.tsx

```diff
@@ -29,28 +30,86 @@ it("renders an image when provided as src", done => {
   };
 
   const { queryByTestId } = render(<FormatFile file={testFile} />);
-  const imageBlockDiv = queryByTestId("imageBlock");
+
+  await waitFor(() => {
+    expect(queryByTestId("internalThumbnailImage")).not.toBeUndefined();
+  });
+
+  const internalThumbnailImage = queryByTestId("internalThumbnailImage");
 
   waitFor(() => {
-    expect(imageBlockDiv).toHaveStyle(`background-image: url(${url})`);
+    expect(internalThumbnailImage).toBeInstanceOf(HTMLImageElement);
     done();
   });
 });
 
-it("it should call the delete handler", () => {
+it("it should call the delete handler", async () => {
   const testFile = {
     key: "234",
     name: "TPS Reports",
     type: "application/pdf",
     size: 1022,
     progress: 1,
+    src: () => Promise.resolve("https://nature/interesting-article.png"),
   };
   const deleteHandler = jest.fn();
-  const { getByLabelText } = render(
-    <FormatFile onDelete={deleteHandler} file={testFile} />,
+  const { getByLabelText, getByText } = render(
+    <body>
+      <FormatFile onDelete={deleteHandler} file={testFile} />
+    </body>,
   );
 
-  fireEvent.click(getByLabelText("Delete"));
+  act(() => {
+    fireEvent.click(getByLabelText("Delete Thumbnail"));
+  });
+
+  await waitFor(() => {
+    expect(
+      getByText("Are you sure you want to delete this file?"),
+    ).toBeInstanceOf(HTMLParagraphElement);
+  });
```

Comment:
This seems to work without the `waitFor` 🤔 

Also, Atlantis can check if the element is in the DOM. Much easier to deal with if that suddenly becomes not a paragraph anymore. We only care about if it's there :D 

```suggestion
  expect(
    getByText("Are you sure you want to delete this file?"),
  ).toBeInTheDocument();
```